src/test/java/edu/cornell/kfs/tax/FormTypes1099Test.java

```diff
@@ -0,0 +1,55 @@
+package edu.cornell.kfs.tax;
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.mchange.util.AssertException;
+
+class FormTypes1099Test {
+
+	@BeforeEach
+	void setUp() throws Exception {
+	}
+
+	@AfterEach
+	void tearDown() throws Exception {
+	}
+
+	@Test
+	void findFormTypes1099FromFormCodeMisc() {
+		FormTypes1099 expectedType = FormTypes1099.MISC;
+		FormTypes1099 actualType = FormTypes1099.findFormTypes1099FromFormCode(FormTypes1099.MISC.formCode);
+		assertEquals(expectedType.formDescription, actualType.formDescription);
```

Comment:
In this test method and the one below it, I would suggest including an assertion that also checks the formCode field.

Reply:
OK